src/js/BookReader.js

```diff
@@ -581,6 +572,25 @@ BookReader.prototype.drawLeafs = function() {
   }
 };
 
+/**
+ * @private
+ */
+BookReader.prototype._createPageContainer = function(index, styles) {
```

Comment:
We can start here. 👍 
I would like to extend the notion further and create a `Page` class when we touch this code next